ui/src/app/views/settings/component.spec.tsx

```diff
@@ -0,0 +1,46 @@
+import { mount } from 'enzyme';
+import * as React from 'react';
+
+import {SettingsReact, SettingsState} from './component';
+
+import {Cluster, ClusterStatus} from 'generated/fetch/api';
+
+describe('SettingsComponent', () => {
+  let props: {};
+
+  const component = () => {
+    return mount<SettingsReact, {}, SettingsState>
+    (<SettingsReact {...props}/>);
+  };
+
+  beforeAll(() => {
```

Comment:
why do you do this before tests?

Reply:
This is necessary to provide the modals a place to live. This should actually be done as part of a global setup for all tests, but I am not entirely sure where that is. If you know, let me know and I will move it?

Reply:
I made an angular helper for this: https://github.com/all-of-us/workbench/blob/bddb11471e19d06cef7a7e46a54581d17f68605d/ui/src/testing/test-helpers.ts#L8

But I think global test setup might be a better place for it going forward with jest. I'd add it there if it's easy to do.

Reply:
I am not sure if that should go here:
https://github.com/all-of-us/workbench/blob/master/ui/test-setup.js#L10
Or here: 
https://github.com/all-of-us/workbench/blob/master/ui/test-shim.js#L10

I think in the setup file? But I don't know if that actually works. I can't find anything that actually does what we want here.

Reply:
I think test-setup is probably an OK place. If that doesn't work, you could alternatively try to use a new file with this different jest config property: https://jestjs.io/docs/en/configuration.html#setuptestframeworkscriptfile-string